Drv/LinuxSerialDriver/CMakeLists.txt

```diff
@@ -5,14 +5,14 @@
 # MOD_DEPS: (optional) module dependencies
 #
 ####
-if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux" OR ${CMAKE_SYSTEM_NAME} STREQUAL "arm-linux-gnueabihf" OR ${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux" OR ${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
```

Comment:
Does the linux serial driver work on Mac OS?

Reply:
Yes!